tests/stem.js

```diff
@@ -37,4 +37,43 @@ test('Lemmatize non word', function(done) {
         myStem.stop();
         done();
     });
+});
+
+test('Fetch all grammemes known word', function(done) {
+    var myStem = new MyStem();
+    myStem.start();
+
+    myStem.fetchAllGrammemes("немцы").then(function(lemma) {
+        /*
+        Существительное, мужской род, одушевлённое, именительный падеж, множественное число
+         */
+        assert.deepEqual( lemma, ['немец', 'S', 'm', 'anim=nom', 'pl' ]);
+    }).then(function() {
+        myStem.stop();
+        done();
+    });
+});
+
+test('Fetch all grammemes non word', function(done) {
+    var myStem = new MyStem();
+    myStem.start();
+
+    myStem.fetchAllGrammemes("шоп78шол").then(function(lemma) {
+        assert.equal( lemma, "шоп78шол");
+    }).then(function() {
+        myStem.stop();
+        done();
+    });
+});
+
+test('Fetch all grammemes unknown word', function(done) {
+    var myStem = new MyStem();
+    myStem.start();
+
+    myStem.fetchAllGrammemes("хелоу").then(function(lemma) {
+        assert.equal( lemma[0], "хелоу");
```

Comment:
It can be useful just to show that we do not return empty elements for the rest of the grammes in the case with the unknown word. But, I agree, that it is not critical at all